lib/bgm.dart

```diff
@@ -43,51 +43,48 @@ class Bgm extends WidgetsBindingObserver {
   ///
   /// It is safe to call this function even when a current BGM track is
   /// playing.
-  void play(String filename, {double volume}) async {
+  Future<void> play(String filename, {double volume}) async {
```

Comment:
just making all methods consistent (async) and allowing callers to await if they want. this should not be breaking.